deploy/gk-deploy

```diff
@@ -31,11 +31,12 @@ SKIP_PREREQ=0
 LOAD=0
 ADMIN_KEY=""
 USER_KEY=""
+GLUSTER_LABEL=""
 
 usage() {
   echo -e "USAGE: ${PROG} [-ghvy] [-c CLI] [-t <TEMPLATES>] [-n NAMESPACE] [-w <SECONDS>]
        [--load] [--admin-key <ADMIN_KEY>] [--user-key <USER_KEY>] [-l <LOG_FILE>]
-       [<TOPOLOGY>]\n"
+       [--gluster-label <GLUSTER_LABEL> ]  [<TOPOLOGY>]\n"
```

Comment:
May be 'node-selector' or similar would be a good name for this option. Gluster label is bit confusing as per me.